### DISCUSSION TITLE
WebviewWindow Browser compatibility

### DISCUSSION THREAD

**Discussion by gladenko** [tauri-apps/tauri#5569] [Q&A] [answered]
I try to open gmail with new WebviewWindow but the render is old style.

New style of gmail not working error is

> Unrecognized Content-Security-Policy directive 'require-trusted-types-for'.

How i manipulate website or webapp or single page application from external url

Thank you in advance!

**Comment by FabianLars** [comment, marked as answer]
`require-trusted-types-for` is not support by webkit/Safari and therefor no-op on Linux and macOS (in the context of Tauri apps).
